app/client/src/sagas/helper.ts

```
@@ -324,3 +326,75 @@ export function* getInitialActionPayload(
     actionConfiguration: actionConfig.actionConfiguration,
   };
 }
+
+export function* getUpdatedTabs(newId: string, currentTabs: string[]) {
+  if (currentTabs.includes(newId)) return currentTabs;
+
+  const newTabs = [...currentTabs, newId];
+
+  return newTabs;
+}
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)